std_msgs/Header header
uint16 robot_id
string robot_state